apiserver/README.md

```diff
@@ -2,21 +2,20 @@
 
 The KubeRay APIServer provides gRPC and HTTP APIs to manage KubeRay resources.
 
-**Note**
+## Note
 
-    The KubeRay APIServer is an optional component. It provides a layer of simplified
-    configuration for KubeRay resources. The KubeRay API server is used internally
-    by some organizations to back user interfaces for KubeRay resource management.
+```text
+The KubeRay APIServer is an optional component. It provides a layer of simplified configuration for KubeRay resources. The KubeRay API server is used internally by some organizations to back user interfaces for KubeRay resource management.
 
-    The KubeRay APIServer is community-managed and is not officially endorsed by the
-    Ray maintainers. At this time, the only officially supported methods for
-    managing KubeRay resources are
+The KubeRay APIServer is community-managed and is not officially endorsed by the Ray maintainers. At this time, the only officially supported methods for
+managing KubeRay resources are:
 
-    - Direct management of KubeRay custom resources via kubectl, kustomize, and Kubernetes language clients.
-    - Helm charts.
+- Direct management of KubeRay custom resources via kubectl, kustomize, and Kubernetes language clients.
+- Helm charts.
 
-    KubeRay APIServer maintainer contacts (GitHub handles):
-    @Jeffwan @scarlet25151
+KubeRay APIServer maintainer contacts (GitHub handles):
```

Comment:
This doesn't seem accurate to me, though I guess the change is really just a formatting fix

Reply:
Yes, it is just a formatting change. We can discuss updating the code ownership at a later time.